functionaltests/src/test/java/specs/paymentmethods/paydirekt/AuthorizationFixture.java

```diff
@@ -0,0 +1,13 @@
+package specs.paymentmethods.paydirekt;
+
+import org.concordion.api.FullOGNL;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@FullOGNL // required by containsSubstring() for redirect URL matching
+public class AuthorizationFixture extends PaydirektFixture {
+
+    // see base PaydirektFixture for implementation
+
+}
```

Comment:
these classes "mocks" are needed for Concordion test framework, because it searches fixtures by class names, but  the logic is common for both _authorization_ and _charge_ transactions (below)